posts/2019-02-20-framework-release-v138-websockets.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Serverless Framework v1.38 - Introducing Websockets Support"
+description: "Check out the latest Serverless Framework v1.38 release."
+date: 2019-02-07
+thumbnail: 'https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework-updates/framework-v137-thumb.png'
+heroImage: 'https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework-updates/framework-v137-header.png'
+category:
+  - news
+authors:
+  - EslamHefnawy
+---
+
+### Introducing Websockets Support
+After years of waiting, [AWS has finally added support for websockets in API Gateway & Lambda](https://aws.amazon.com/blogs/compute/announcing-websocket-apis-in-amazon-api-gateway/), and because CloudFormation didn't have support for websockets at the time of the initial release, we've created an [external plugin](https://github.com/serverless/serverless-websockets-plugin) that lets you integrate websockets into you Serverless Framework projects.
+
+Fast forward couple of weeks, [CloudFormation now has official support for Websockets](https://aws.amazon.com/about-aws/whats-new/2019/02/automate-websocket-api-creation-in-api-gateway-with-cloudformation/). With that in place, we immediately started adding support for Websockets in the framework core. We're now excited to announce that this 1.38.0 release includes support for Websockets.
+
+**Note:** Now that we have official support for websockets in core, the [serverless-websockets-plugin](https://github.com/serverless/serverless-websockets-plugin) is officially deprecated, and we will no longer maintain it.
```

Comment:
Do we have a grace period or is will this be deprecated immediately?

Reply:
immediately